src/openproduct/locaties/models/locatie.py

```diff
@@ -56,6 +56,7 @@ class Locatie(BaseLocatie):
     class Meta:
         verbose_name = _("Locatie")
         verbose_name_plural = _("Locaties")
+        ordering = ("id",)
```

Comment:
Usually we order by `-id` or `-pk` to show newest first, but if this was agreed upon, this is fine I think